src/transaction/log_record.hpp

```diff
@@ -37,33 +37,19 @@ enum log_rectype
   /* In order of likely of appearance in the log */
   LOG_SMALLER_LOGREC_TYPE = 0,	/* A lower bound check */
 
-#if 0
```

Comment:
Please leave this file as before. The 'Obsolete' and other comments on each member of enum are information for developers.